### ISSUE TITLE
access HXN's new mongo database

### ISSUE THREAD

**Issue by leofang** [NSLS-II/PyXRF#139]
Related: NSLS-II-HXN/ipython_ophyd/pull/15, NSLS-II/playbooks/pull/188, leofang/ptycho_gui/issues/24

The following lines
https://github.com/NSLS-II/PyXRF/blob/9f74bbd1aa4e7bdf3c9085bef77370c702cb6aac/pyxrf/db_config/hxn_db_config.py#L6-L26
will need to be replaced by
```python
# new mongo database
_mds_config = {'host': 'mongodb://xf03id1-mdb01:27027,xf03id1-mdb02:27027,xf03id1-mdb03:27027/?replicaSet=mongors',
               'port': 27027,
               'database': 'datastore-2',
               'timezone': 'US/Eastern'}
_fs_config  = {'host': 'mongodb://xf03id1-mdb01:27027,xf03id1-mdb02:27027,xf03id1-mdb03:27027/?replicaSet=mongors',
               'port': 27027,
               'database': 'filestore-2'}
db = Broker(MDS(_mds_config), FileStore(_fs_config))
```
Not sure what else needs to be changed...

**Comment by tacaswell** [comment]
Can you please use `db = Broker.named()` instead?

**Comment by licode** [comment]
I think I already used Broker.named() here. sure, will confirm that.

**Comment by danielballan** [comment]
Looks like `Broker.named` is applied in this open PR: https://github.com/NSLS-II/PyXRF/pull/140/files#diff-98899c46bfc2732e522ad2dee8d386b1R3

**Comment by leofang** [comment]
closed by #140
